packages/pebble-web/src/components/typings/DateInput.ts

```diff
@@ -11,6 +11,7 @@ export interface DateInputProps {
   disabled?: boolean;
   placement?: Placement;
   wrapperClassName?: string;
+  dropDownWrapperClassName?:string
```

Comment:
when overriding internal styles, prefer using target className as prop

```suggestion
  dropDownClassName?:string
```